.github/workflows/changelog.yml

```
@@ -0,0 +1,33 @@
+name: Changelog check
```

Always use active verbs in workflow names as they will be displayed in the UI 🙂 

```suggestion
name: Check changelog
```